eb_hooks.py

```diff
@@ -376,6 +376,18 @@ def pre_configure_hook_atspi2core_filter_ld_library_path(self, *args, **kwargs):
         raise EasyBuildError("at-spi2-core-specific hook triggered for non-at-spi2-core easyconfig?!")
 
 
+def pre_configure_hook_highway_disable_tests(self, *args, **kwargs):
+    """
+    pre-configure hook for Highway: disable tests on neoverse_v1
+    cfr. https://github.com/EESSI/software-layer/issues/469
+    """
+    cpu_target = get_eessi_envvar('EESSI_SOFTWARE_SUBDIR')
+    if self.name == 'Highway' and cpu_target == CPU_TARGET_NEOVERSE_V1:
```

Comment:
From the Highway issue, this is most likely a compiler bug with GCC, should we also restrict the hook based on the compiler and version?

Reply:
Done!